test.sh

```diff
@@ -7,7 +7,7 @@ die() { set +v; echo "$*" 1>&2 ; sleep 1; exit 1; }
 
 server_up() {
   TRIES=0
-  MAX_TRIES=60
+  MAX_TRIES=500
```

Comment:
If you're waiting 9 minutes for the server to come up, something is really wrong. (The 60 seems over-generous, actually.)

Reply:
The issue I was having was that the build step seems considerably longer now because of `vitessce`'s dependencies (all of the gl libraries, higlass etc.).  I'll tone this down though.